lib/viewing/CloseableView.js

```diff
@@ -9,12 +8,28 @@ var CloseableView = {
     onClose: noop,
 
     close: function() {
-        triggerOnCloseAndUnbindEventsOn(this);
+        var onCloseError;
+
+        this.trigger("close");
+
+        onCloseError = ifOnCloseError(this);
```

Comment:
this is ugly because we need to keep the order of execution and the attempts to close resources but we still want to rethrow the error